build-scripts/android/build.sh

```diff
@@ -2,11 +2,11 @@
 
 set -e
 cd android
-./gradlew check
+./gradlew app:check
 
 if [[ $BUILD_RELEASE == 1 ]]; then
     ./gradlew assembleRelease --info --console plain | tee
 else
-    ./gradlew build --info --console plain | tee
```

Comment:
I see travis is now green, are we skipping all the lint checks with this? I am afraid that we have removed many tests and if we skip also the lint we will not have many alerts when landing new code but we could live with this for landing this PR and work on the tests and the lint issue later on in a follow up